### PR TITLE
Populate channel name in Japanese emails

### LINKED ISSUE #3075: Channel name is not populated in channel transfer e-mails
## Description

When a channel transfer request is made, Creators receive various e-mails concerning the channel transfer. However, the channel name is not being populated in the e-mail and appears as `%{channel_name}`. 

**For example:**

image

### PR DESCRIPTION
**Pull request by tsmartt**
Resolves #3075. We were using a multibyte percentage symbol for
variable substitution which did not trigger the substitution.
This has been corrected in multiple emails as well as updating
the preview code to correctly preview the emails in Japanese.

#### Pull Request Checklist

- [X] Adequate test coverage exists to prevent regressions -- [Guide to Testing](https://guides.rubyonrails.org/testing.html)
- [ ] XSS is mitigated -- [Guide to XSS Prevention](https://guides.rubyonrails.org/security.html#cross-site-scripting-xss)
- [X] No raw SQL -- [Guide to SQL Injection Prevention](https://guides.rubyonrails.org/security.html#sql-injection)
- [ ] UI/UX is responsive -- [Guide to Responsive UI/UX](https://developers.google.com/web/fundamentals/design-and-ux/responsive/)
- [ ] Integrated Matomo for new UI elements -- [Guide to Matomo](https://developer.matomo.org/guides/integrate-introduction)
- [ ] Passes checklist for Progressive Web App -- [Guide to PWAs](https://developers.google.com/web/progressive-web-apps/checklist)

### CODE REVIEW

File: test/fixtures/publisher_notes.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ thread_note:
 
 child_note:
   publisher: just_notes
-  note: "This is child note"
+  note: "This is child note@"
```
Comment (yachtcaptain23) [inline test/fixtures/publisher_notes.yml:40]: Unused change?

File: test/mailers/previews/publisher_mailer_preview.rb [hunk 2]
```diff
@@ -85,10 +86,6 @@ def email_user_on_hold
     PublisherMailer.email_user_on_hold(Publisher.first)
   end
 
-  def update_to_tos
```
Comment (yachtcaptain23) [inline test/mailers/previews/publisher_mailer_preview.rb:88]: I think if you want to remove this, you should also remove `lib/tasks/email_tos_update.rake` and `update_to_tos` from `config/locales/ja.yml`.

File: test/fixtures/publisher_notes.yml (hunk 1 above)
Comment (tsmartt) [reply test/fixtures/publisher_notes.yml:40]: No, this was related to one of the previewers which required a @ symbol.

File: test/mailers/previews/publisher_mailer_preview.rb (hunk 2 above)
Comment (tsmartt) [reply test/mailers/previews/publisher_mailer_preview.rb:88]: Is this used anymore? This was going to be a followup pr, but I'll include it in this one if it's not used.

Comment (yachtcaptain23) [reply test/mailers/previews/publisher_mailer_preview.rb:88]: The `update_to_tos` was a one-off script to send a one-off email. We don't send one-off emails from our servers anymore as we just send via SendGrid website instead.

Comment (tsmartt) [reply test/mailers/previews/publisher_mailer_preview.rb]: I restored it to limit this PR scope. Will issue a followup.

**Review by yachtcaptain23 — APPROVED** [review]

**Pull request merged**